iot/mqttsnippets/create_jwt.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Google LLC. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package mqttsnippets
+
+// [START iot_mqtt_jwt]
+
+import (
+	"errors"
+	"io/ioutil"
+	"time"
+
+	jwt "github.com/dgrijalva/jwt-go"
+)
+
+// createJWT creates a Cloud IoT Core JWT for the given project id.
+// algorithm can be one of ["RSA256", "ES256"]
+func createJWT(projectID string, privateKeyPath string, algorithm string, expiresMin int) (string, error) {
```

Comment:
Make expiresMin a `time.Duration` and use `time.Now().Add(expiration).Unix()`? Alternatively, make the expiration the `time.Time` it should expire?